openqabot/amqp.py

```diff
@@ -94,6 +98,3 @@ def handle_incident(self, inc_nr: int, message) -> None:
         # Try to approve incident
         approve = Approver(self.args, inc_nr)
         approve()
-
-    def handle_aggregate(self, unused_build: str, unused_message) -> None:
-        return
```

Comment:
Sometimes it can be useful to have a function within a `class` even if `self` isn't used, e.g. to be able to override it in a derived class or if `self` might be used later. And here most likely the latter is the case. So I would avoid this change. (And your version also prompts the question: Why is `handle_incident` part of the `class` and `handle_aggregate` not? Both functions will be similar once fully implemented. So this is not very consistent.)